.azure-perf-ci.yml

```diff
@@ -0,0 +1,32 @@
+trigger:
```

Comment:
Can you call this file `performance.yml` and put it under `eng/pipelines/`? something to what we did for `outerloop.yml`.